packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -1716,21 +1679,26 @@ void main() {
   });
 
   testUsingContext(
-    'should show --ios-language deprecation error for non-plugin templates',
+    'should show --ios-language deprecation warning for all templates',
     () async {
       final command = CreateCommand();
       final CommandRunner<void> runner = createTestCommandRunner(command);
 
-      await expectToolExitLater(
-        runner.run(<String>['create', '--no-pub', '--ios-language=swift', projectDir.path]),
-        contains('The "ios-language" option is only supported for "--template=plugin".'),
+      await runner.run(<String>['create', '--no-pub', '--ios-language=swift', projectDir.path]);
+      expect(
+        logger.warningText,
+        contains(
+          'The "--ios-language" option is deprecated and no longer has any effect. '
+          'Swift is always used for iOS-specific code in plugins. '
+          'This flag will be removed in a future version of Flutter.',
+        ),
       );
     },
     overrides: {FeatureFlags: () => TestFeatureFlags(), Logger: () => logger},
   );
 
   testUsingContext(
-    'should show --ios-language deprecation warning issue for Objective-C plugins',
+    'should show --ios-language deprecation warning for Objective-C plugins',
```

Comment:
This test can be deleted since there's no longer any branching logic for plugins, it's going to warn for all templates.

Reply:
done